aries/aries_vcx_core/Cargo.toml

```diff
@@ -18,7 +18,7 @@ agency_client = { path = "../misc/legacy/agency_client" }
 indy-vdr = { git = "https://github.com/hyperledger/indy-vdr.git", rev = "c143268", default-features = false, features = ["log"] }
 indy-credx = { git = "https://github.com/hyperledger/indy-shared-rs", tag = "v1.1.0", optional = true }
 # anoncreds = { git = "https://github.com/hyperledger/anoncreds-rs", tag = "v0.2.0-dev.5", optional = true }
-anoncreds = { git = "https://github.com/mirgee/anoncreds-rs.git", rev = "9a7539c", optional = true }
+anoncreds = { git = "https://github.com/mirgee/anoncreds-rs.git", rev = "a85d739", optional = true }
```

Comment:
Did I guess this one correctly? https://github.com/mirgee/anoncreds-rs/pull/1/commits/a85d739c10a90834dbd912508cbceba77b114b64

https://github.com/hyperledger/aries-vcx/actions/runs/7887341121/job/21522371585?pr=1123

Reply:
I don't understand what is being guessed and why, and why the revision should change, but the diff seems correct.

Reply:
The commit hash which should be used as a dependency `rev` is being guessed because commit `9a7539c` does not seem to exist and that causes the CI failures as per link above.

Reply:
I have seen the error you linked, of course, and your comment unfortunately is just restating it without adding any additional information.

Reply:
Neither do I.

<img width="805" alt="Screenshot 2024-02-14 at 7 52 13" src="https://github.com/hyperledger/aries-vcx/assets/2664090/367a7890-ea42-4f80-af38-6048e91d4328">

Reply:
Interesting, I wonder how come the CI was passing. Perhaps it was some-prebase revision which happened to be cached somewhere.